tests/qos/files/cisco/qos_param_generator.py

```diff
@@ -71,6 +71,9 @@ def __init__(self, qos_params, duthost, dutAsic, topo, bufferConfig, portSpeedCa
                 self.reduced_pause_thr = 3 * (1024 ** 2)
             else:
                 self.reduced_pause_thr = 2.25 * (1024 ** 2)
+            if dutAsic == "gr":
+                self.reduced_pause_thr = self.gr_get_hw_thr_buffs(self.reduced_pause_thr
```

Comment:
This function is just a variety of rounding and shifting operations, and does not read any hardware-related data from the ASIC. Can it be considered that it is just the accuracy relationship that causes some rounding of the hardware value and the difference in software configuration?

Reply:
Yes, there is hardware rounding.